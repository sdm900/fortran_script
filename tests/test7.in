do i=1,10
print "i, is", i
enddo

