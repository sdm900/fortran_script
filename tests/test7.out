i, is     1.00000 
i, is     2.00000 
i, is     3.00000 
i, is     4.00000 
i, is     5.00000 
i, is     6.00000 
i, is     7.00000 
i, is     8.00000 
i, is     9.00000 
i, is    10.00000 
